src/braket/experimental/autoqasm/README.md

```diff
@@ -59,18 +59,16 @@ AutoQASM enables users to use more complicated program constructs with a compact
 structure. We can demonstrate this with an active reset program:
 ```
 # A program that actively resets a qubit back to the ground state
-@aq.function
+@aq.function(num_qubits=1)
```

Comment:
This program does not require `num_qubits` to be specified at all. The `qubit=0` argument is evaluated by the Python interpreter at trace time, and so autoqasm correctly infers that only qubit 0 is used.

It may still be fine to keep this here for instructional reasons, just wanted to mention it.

Reply:
Ah yes I can remove it. Actually Tim already replaced this whole section so it'll be short lived either way